even_fib_nums.js

```diff
@@ -1,18 +1,36 @@
+/*jshint esversion: 6 */
 /**
  * Return the total sum of all even fibonacci numbers up to and including
  * the value given to you at `maxFibValue`
  *
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
-  var sum = 0;
 
-  // do your work here
 
+ function _sumFibs( maxFibValue ) {
+
+  var sum = 0;
+
+  function isFib(n) {
```

Comment:
Something to keep in mind when using a solution like this is to keep your code "readable." The idea behind that is, if someone else was to come in after you and "take over" this code base, would they understand what you're trying to do. Given that we talked about this in class, I would suggest a comment above this function about how this is using a mathematical approach to figuring out the problem and also include the url of the webpage where you got this formula.